readme.md

```diff
@@ -0,0 +1,71 @@
+Misc-deps is a collection of various static dependencies used by [NVDA](https://github.com/nvaccess/nvda).
+
+See also [NVDA git submodules documentation](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/createDevEnvironment.md#git-submodules) for context.
+
+# Updating contents
+
+## include
+
+### Adobe Acrobat Access
+
+1. Download the latest [Adobe Acrobat SDK](https://developer.adobe.com/console/servicesandapis)
+1. Extract the zip
+1. Copy the file from `Adobe\Acrobat DC SDK\Version 1\AccessibilitySupport\AcrobatAccess.idl`
+
+### iSimpleDOM
+
+MSAA interfaces for gecko / firefox.

```

Comment:
Might be more correct to call them ISimpleDOM interfaces. You get to them via QueryService from an IAccessible, but they really are their own custom interfaces.
Also it is worth noting, that you only need ISimpleDOM.idl, ISimpleDOMNode.idl, iSimpleDOMDocument.idl and iSimpleDOMText.idl. 
I did a quick check, and those 4 files are self-containing - so I'm pretty sure like before they don't import or rely on any other files in that directory.